scripts/analytics/user_summary.py

```diff
@@ -6,12 +6,36 @@
 from modularodm import Q
 
 from website.app import init_app
-from website.models import User
+from website.models import User, NodeLog
 from scripts.analytics.base import SummaryAnalytics
 
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.INFO)
 
+LOG_THRESHOLD = 11
+
+
+# Modified from scripts/analytics/depth_users.py
+def count_user_logs(user):
+    logs = NodeLog.find(Q('user', 'eq', user._id))
+    length = logs.count()
+    if length > 0:
+        item = logs[0]
+        if item.action == 'project_created' and item.node.is_bookmark_collection:
+            length -= 1
+    return length
+
+
+# Modified from scripts/analytics/depth_users.py
+def get_number_of_depth_users(active_users):
```

Comment:
For the analytics requirements ticket you have on the side, make sure there's something in the requirements that ensures that it will work for user analytics as well. Iterating over 40k users every night is less than ideal, hence my uncharacteristic optimization above. Once we have the caching, we can eliminate most of the work from this task.